src/instance.ts

```diff
@@ -576,7 +745,7 @@ class Instance extends common.ServiceObject {
    *   const apiResponse = data[1];
    * });
    */
-  getMetadata(callback) {
+  getMetadata(callback: GetInstanceMetadataCallback): Metadata {
```

Comment:
This looks wrong

- I think we want to use `IInstance` over `Metadata` (AFAIK `Metadata` is an alias to `any`). 
- Current signature implies that this is a synchronous method, but it is not
- Missing want a Promise overload